sdcm/nemesis.py

```diff
@@ -2609,6 +2609,9 @@ def is_streaming_started():
                             node=self.target_node), \
             DbEventsFilter(db_event=DatabaseLogEvent.RUNTIME_ERROR,
                            line="Fail to send STREAM_MUTATION_DONE",
+                           node=self.target_node), \
+            DbEventsFilter(db_event=DatabaseLogEvent.DATABASE_ERROR,
+                           line="rpc stream id",
```

Comment:
I don't think this part of the line is specific enough.

Reply:
the error is "stream id x not found"

Reply:
@roydahan , fixed.